### DISCUSSION TITLE
How can i update REI?

### DISCUSSION THREAD

**Discussion by Scorpions95** [shedaniel/RoughlyEnoughItems#996] [Q & A] [answered]
Want to update REI, but how?

**Comment by shedaniel** [comment, marked as answer]
What? Just download the latest version.

**Reply by Scorpions95** [reply]
i did but i see these errors
https://i.imgur.com/jxTDfgX.png

**Comment by Scorpions95** [comment]
where i can download latest version?

**Reply by shedaniel** [reply]
CurseForge

**Reply by Scorpions95** [reply]
> CurseForge

yeah, i donwload rei from there
but there it error
https://i.imgur.com/Bfo2tKf.png
[latest.log](https://github.com/shedaniel/RoughlyEnoughItems/files/9175527/latest.log)

**Reply by shedaniel** [reply]
According to EULA we do not support TLauncher, I am locking this discussion. This is a violation as providing support can also harm the project, please understand
